www/README.md

```diff
@@ -37,55 +37,7 @@ In some cases, you may need to install `gawk` in order to obtain the `ffi` gem.
 
 ## How-To: Deploy
 
-1. [Setup your workstation](#setup)
-2. Configure your environment
-
-  * AWS_BUCKET - for production deploys, this should be `habitat-www-live`, for acceptance deploys `habitat-www-acceptance`
-  * AWS_DEFAULT_REGION - this should be `us-west-2`
-  * AWS_ACCESS_KEY_ID - your personal AWS account identifier
-  * AWS_SECRET_ACCESS_KEY - your AWS account's personal access key
-  * FASTLY_API_KEY - your Fastly account's personal access key
-  * FASTLY_SERVICE_KEY - The Fastly Service ID for either `www.habitat.sh` or `www.acceptance.habitat.sh`, depending on which one you're deploying.
-
-  ```
-  $ export AWS_ACCESS_KEY_ID=AKIBJTPWS4EK8L2RXUBZ
-  ```
-
-  > Note: values for each of these can be found by logging into your AWS and Fastly control panels.
-  > Note: make sure that the AWS credentials you use are for the Habitat AWS account and not the regular Chef AWS account.
-
-### A slight wrinkle
-If you are a Chef employee, you might be (should be) using [okta_aws](https://github.com/chef/okta_aws).
-
-In this case, you will need to run the following:
-
-```sh
-okta_aws habitat
-export AWS_DEFAULT_PROFILE=habitat
-```
-
-This eliminates the need to set `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
-manually. The other variables still need to be set.
-
-1. Change to the `www` directory
-
-   ```
-   $ cd www
-   ```
-
-1. Run the deploy make task
-
-    ```
-    $ make deploy
-    ```
-
-    > Note: If the above task fails with a 403 Forbidden error, and you're
-    > a Chef employee using okta_aws, you can deploy the web site an alternate
-    > way.
-
-    ```
-    make build
-    cd build
-    aws s3 sync . s3://$AWS_BUCKET
-    make purge_cache
-    ```
+This happens automatically now as part of our CI pipeline. When a PR is opened,
+if files inside the `www` directory were modified, then the web site will be
+deployed to acceptance. When a PR merges to master, the same check is done, and
+the web site will be deployed to production, if necessary.
```

Comment:
Where is the code that runs this on PRs? I didn't see that.

I think that automatically deploying to acceptance with every PR would open us up to race conditions (though likely not very often, as there aren't often multiple people actively working on different website-affecting changes).

Maybe adding a step to the verify pipeline that ensured that the website could at least _build_? Then people could manually deploy to acceptance as a final pre-merge task?

Reply:
I wrote this blurb when I first started this PR and forgot to update it.  There is no code to automatically deploy the web site to acceptance when a PR opens, per our discussion about the security hole that would open.

Reply:
Updated in https://github.com/habitat-sh/habitat/pull/7273/commits/fcc0d27728ebbccd9bf5d448bf56339da9861f49